profiles/kentik_snmp/cisco/cisco-asr.yml

```diff
@@ -1,5 +1,6 @@
 # https://www.circitor.fr/Mibs/Html/C/CISCO-PROCESS-MIB.php
 # http://www.circitor.fr/Mibs/Html/C/CISCO-MEMORY-POOL-MIB.php
+# https://www.circitor.fr/Mibs/Html/C/CISCO-PROCESS-MIB.php
```

Comment:
this looks like the same link on line 1? is it needed?

Reply:
No it should be CISCO-PRODUCT-MIB.php.  Shit.  I will close and resubmit.

Reply:
@afsrwin you can just update and add a new commit to your branch and it will automatically update this PR; for future reference